### DISCUSSION TITLE
Borrow missing from the tools directory

### DISCUSSION THREAD

**Discussion by StefanieBond** [ArctosDB/arctos#6916] [General]
For documenting an incoming loan, under transactions there was a search 'borrow' and create 'borrow'.  Now it is all folded under 'Search loans', and you can select type of loan 'borrow' and then you can 'create borrow'.  This works, but I wonder why the 'borrow button' up front disappeared.

**Comment by dustymc** [comment]
Moved to Issue https://github.com/ArctosDB/arctos/issues/6917, this is actionable, closing.
